package.json

```diff
@@ -224,7 +224,8 @@
     "react-custom-scrollbars": "4.2.1",
     "react-datetime": "3.0.4",
     "react-dom": "16.13.1",
-    "react-intl": "2.7.2",
+    "react-intl": "2.9.0",
+    "react-intl-hooks": "^1.0.11",
```

Comment:
In general we avoid updating dependencies within feature PRs if not necessary - we don't know if this change might cause unrelated trouble somewhere in the app. If this change is not required, please move it to its own PR which can be tested and approved in isolation.

Reply:
Thank you for the tip @DominikGuzei ! I will use injectIntl. No need for extra dependencies